dev/io.openliberty.transport.ssl_fat/fat/src/io/openliberty/transport/ssl/SSLOptionsTest.java

```diff
@@ -0,0 +1,341 @@
+    private static final String DEFAULT_SSLOPTIONS_CONFIG = "defaultSSLOptions.xml";
+    private static final String SSLOPTIONS_CONFIG = "SSLOptions.xml";
+
+    private static final String DEFAULT_REALM = "Basic Authentication";
+    private static final String DEFAULT_SERVLET_NAME = "ServletName: BasicAuthServlet";
+    private static final String DEFAULT_CONTEXT_ROOT = "/basicauth";
+
+    private static final String KEYSTORE = "sslOptions.jks";
+    private static final String TRUSTSTORE = "trust.jks";
+    private static final String ALTERNATE_TRUSTSTORE = "optionsTrust.jks";
+    private static final String PASSWORD = "Liberty";
+
+    @Server("SSLOptionsServer")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+
+        /*
+         * Transform apps for EE9+.
+         */
+        if (JakartaEEAction.isEE9OrLaterActive()) {
+            JakartaEEAction.transformApp(Paths.get(server.getServerRoot() + "/apps/basicauth.war"));
+        }
+
+        // The app basicauth is added in the config and we validate it was actually installed
+        server.addInstalledAppForValidation("basicauth");
+
+        server.startServer(SSLOptionsTest.class.getSimpleName() + ".log");
+        // Wait for SSL endpoint to start
+        assertNotNull("We need to wait for the SSL port to open",
+                      server.waitForStringInLog("CWWKO0219I:.*-ssl"));
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        // Ignoring unrelated warning due to app install SRVE0272W: JSP Processor not defined. Skipping : BasicAuthJSP.jsp
+        // Ignore CWWKO0801E because it is expected for handshake failures
+        server.stopServer("SRVE0272W", "CWWKO0801E");
+    }
+
+    /**
+     * Save the server configuration before each test, this should be the default server
+     * configuration.
+     *
+     * @throws Exception
+     */
+    @Before
+    public void beforeTest() throws Exception {
+        server.saveServerConfiguration();
+
+        ServerConfiguration configuration = server.getServerConfiguration();
+        LOG.info("Server configuration that was saved: " + configuration);
+    }
+
+    /**
+     * Restore the server configuration to the default state after each test.
+     *
+     * @throws Exception
+     */
+    @After
+    public void afterTest() throws Exception {
+        // Restore the server to the default state.
+        server.setMarkToEndOfLog();
+        server.restoreServerConfiguration();
+        server.waitForConfigUpdateInLogUsingMark(null);
+    }
+
+    /**
+     * Test that an SSL handshake failure does get logged with the default
+     * sslOptions value suppressHandshakeErrors="false".
+     */
+    @Test
+    public void handshakeFailureGetsLogged() throws Exception {
+        LOG.info("Entering handshakeFailureGetsLogged");
```

Comment:
I'm not seeing a server.xml in the SSLOptionsServer only a bootstrap.properties so what server.xml are we using for this test? I seem to be missing where we are setting the default server.xml

Reply:
It would help I guess if I scrolled down....resolving